### PR TITLE
#166237753 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by tolulope-od**
#### What does this PR do?
Integrates Travis for continuous integration
Changes filenames to follow `camelCase` convention

#### Description of Task to be completed?
Set up Travis to automatically run tests and other preliminary builds

#### How should this be manually tested?
The badge gotten from Travis can be viewed in the README documentation on this branch

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166237753](https://www.pivotaltracker.com/story/show/166237753)

#### Screenshots (if appropriate)
**First Build For Current Branch on Travis**

Screenshot 2019-05-31 at 9 50 51 AM

### CODE REVIEW

**Review by chialuka — APPROVED** [review]
Good job Tolu. Way to go

**Review by ayodejiAA — APPROVED** [review]
Nice job, @tolulope-od. Approved!

**Review by Nerocodes — APPROVED** [review]

**Comment by chiboycalix** [comment]
Nice Job,

**Review by chiboycalix — APPROVED** [review]
Job well done @tolulope-od I love seeing that green color. Approved

File: server/test/undefinedRoute.spec.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ chai.use(chaiHttp);
 const { expect } = chai;
 
 describe('Undefined Routes', () => {
-  it('should return status 404 and an error attribute', done => {
+  it('should return status 404 and an error attribute', (done) => {
```
Comment (abdulfatai360) [inline server/test/undefinedRoute.spec.js:9]: Thank you for catching this error. It slipped my mind. Well done!

**Review by abdulfatai360 — APPROVED** [review]
Nice work in general, Tolu

**Pull request merged**